exchanges/gateio/gateio_types.go

```diff
@@ -25,6 +25,9 @@ const (
 	// Loan sides
 	sideLend   = "lend"
 	sideBorrow = "borrow"
+	// cancel sides
+	sideBid = "bid"
```

Comment:
Then can purge these after the other nits are addressed so there's less duplicates/stuttering of this